.buildkite/docker-compose.yml

```diff
@@ -7,12 +7,16 @@ services:
       context: ./../docker
       args:
         platform: generic
+        BUILDKIT_INLINE_CACHE: 1
     volumes:
       - type: bind
         source: ./..
         target: /home/nubots/NUbots
         read_only: true
         consistency: cached
+    environment:
+      COMPOSE_DOCKER_CLI_BUILD: 1
+      DOCKER_BUILDKIT: 1
```

Comment:
Do these environment variables only apply within the docker image or outside as well?